test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -564,8 +564,8 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
       expect(resp.status).toEqual(200);
       expect(resp.text).toContain(BENEFICIAL_OWNER_TYPE_PAGE_HEADING_SOME_IDENTIFIED);
       expect(resp.text).toContain(ErrorMessages.MUST_ADD_MANAGING_OFFICER);
-      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(1);
-      expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(config.ACTIVE_SUBMISSION_BASE_PATH);
+      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(2);
+      expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(config.BENEFICIAL_OWNER_STATEMENTS_WITH_PARAMS_URL);
```

Comment:
Not quites sure what's happening here. Can you clarify why the URL now refers to a different page? Same with the tests below.

Reply:
As mentioned in the comment above, this is now captured in the `_WITH_PARAMS_URL` assertion